doc/contributing/docs/markup/code.rst

```diff
@@ -1,8 +1,49 @@
 Writing about code
 ==================
 
+When writing articles, you have to format code specially, separating it from
+other text. This document will guide you through typical cases when
+it is recommended to use code highlighting.
+
+Defining what code is
+---------------------
+
+It is not difficult to recognize the code snippet in the text. First of all,
+it is probably code if the expression contains characters that ordinary words
+do not have, such as ``_, {}, [ ], .``.
+Also, you should format the expression as code if it fits at least one
+of the items in the list below:
+
+*   attribute names, values and class names,
+*   constant values for attributes,
+*   method and function names,
+*   language keywords,
+*   data types,
+*   error messages,
+*   URL for localhost and intranet,
```

Comment:
It depends on whether the user is going to open this link or not. For example, in a manual for building and running our docs on a local machine, we have an absolutely clickable http://localhost:8000 link.